docs/index.rst

```diff
@@ -86,8 +84,11 @@ Table of contents:
   :caption: Annexes
   :maxdepth: 1
   
+  getting-started/changelog
```

Comment:
Move the files to the folders, so everything matches.
It'll avoid confusion in the long run

Reply:
moved